pkgs/tools/typesetting/tex/texlive/build-tex-env.nix

```diff
@@ -186,9 +185,7 @@ let
     inherit meta passthru;
   }).overrideAttrs { outputs = [ outName ]; } // { outputSpecified = true; }));
 
-  passthru = lib.optionalAttrs (! __combine) (splitOutputs // {
-    all = builtins.attrValues splitOutputs;
-  }) // {
+  passthru = lib.optionalAttrs (! __combine) splitOutputs // {
```

Comment:
Actually
```suggestion
  passthru = {
```

Reply:
Ok, *now* all the fake multi-output code has been removed: `passthru` contains only attributes that are not also outputs of the main derivation below.